### DISCUSSION TITLE
Disable hashtag suggestions

### DISCUSSION THREAD

**Discussion by fedione** [mastodon/mastodon#27963] [General]
Sometimes it is very hard, to set hashtags in text beginning with a capital letter.
So is there any way to disable auto suggestions?
This would be nice especially for German language user.

**Comment by fedione** [comment]
Any recommendations?

**Comment by weex** [comment]
You can press space bar after creating your hashtag to bypass the autocorrection. May be handled by https://github.com/mastodon/mastodon/issues/16449
